types/eslint/use-at-your-own-risk.d.ts

```diff
@@ -0,0 +1,8 @@
+/** @deprecated */
+export const builtinRules: Map<string, import("./index.js").Rule.RuleModule>;
```

Comment:
should this be this?

```suggestion
import type {Rule} from './index';

export const builtinRules: Map<string, Rule.RuleModule>;
```

Reply:
It's identical in meaning - this is just a bit more compressed, since it's the only import in the file.